utils/test/tx_tracing_helpers.py

```diff
@@ -37,22 +37,30 @@ def display_voting_call_trace(tx: TransactionReceipt) -> None:
 
 
 def count_vote_items_by_events(tx: TransactionReceipt, voting_addr: str) -> int:
-    events = tx_events_from_trace(tx)
+    events = tx_events_from_receipt(tx)
     ev_dict = EventDict(events)
 
     calls_slice = ev_dict["LogScriptCall"]
     return sum(map(lambda x: x["src"] == voting_addr, calls_slice))
 
 
 def display_voting_events(tx: TransactionReceipt) -> None:
-    dict_events = EventDict(tx_events_from_trace(tx))
+    dict_events = EventDict(tx_events_from_receipt(tx))
```

Comment:
The `display_voting_events()` result will differ when used with `tx_events_from_receipt()`. The `tx_events_from_receiptl()` doesn't contain information to show the implementations. Here is an example of the difference:

- When used with `tx_events_from_trace(tx)`
![image](https://github.com/user-attachments/assets/0d6cf1a0-f963-40a0-a369-08d722790dd5)

- When used with `tx_events_from_receipt(tx)`:
![image](https://github.com/user-attachments/assets/34b5af7e-5605-486d-8419-789514123542)